src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6841,6 +6841,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureCollectionExpressions" xml:space="preserve">
     <value>collection expressions</value>
   </data>
+  <data name="IDS_FeatureCollectionExpressionArguments" xml:space="preserve">
+    <value>collection expression arguments</value>
+  </data>
```

Comment:
Grrrr

Reply:
Yes, these two resource strings are essentially the ones you added and removed from [https://github.com/dotnet/roslyn/pull/76003](https://github.com/dotnet/roslyn/pull/76003).